sentry/src/main/java/io/sentry/Breadcrumb.java

```diff
@@ -667,11 +667,7 @@ public void setUnknown(@Nullable Map<String, Object> unknown) {
   @Override
   @SuppressWarnings("JavaUtilDate")
   public int compareTo(@NotNull Breadcrumb o) {
-    int timestampCompare = timestamp.compareTo(o.timestamp);
-    if (timestampCompare == 0) {
-      return nanos.compareTo(o.nanos);
-    }
-    return timestampCompare;
+    return nanos.compareTo(o.nanos);
```

Comment:
We're now solely relying on `nanos` for ordering breadcrumbs which should avoid an unnecessary comparison on `timestamp`. If we have to create Breadcrumbs after the fact (e.g. because they're attached to an OTel span and we want to mirror them in Sentry), we'll have to pass `nanos` into the ctor.